packages/modules/link-modules/src/utils/generate-schema.ts

```diff
@@ -32,11 +32,7 @@ export function generateGraphQLSchema(
 
   for (const extend of joinerConfig.extends ?? []) {
     const extendedModule = MedusaModule.getModuleInstance(extend.serviceName)
-    if (
-      !isReadOnlyLink &&
-      !extendedModule &&
-      !extend.relationship.isInternalService
-    ) {
+    if (!extendedModule) {
```

Comment:
why do we changed that?

Reply:
This is more a safe guard for us. In practice this condition will never happen because the read only links were already filtered against loaded modules.